react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,37 +1,55 @@
-import React, { Component } from 'react';
+import React from 'react';
 
 import Square from '../Square';
 
 import styles from './styles.module.scss';
 
-class Board extends Component {
-  renderSquare(i) {
-    return <Square value={i}/>;
-  }
-
-  render() {
-    const status = 'Next player: X';
-    return (
+const Board = (props) => 
```

Comment:
As a convention we follow, functional components should be defined as regular functions, not arrow ones.
Also, apply destructuring on props to avoid repeating `props. ...` 
```js
function Board({ prop1, prop2, ... }) {
  return ...
}
```